src/utils/buttonSizeClassName.tsx

```diff
@@ -0,0 +1,4 @@
+export function buttonSizeClassName(buttonSize: string | null) {
+  if (buttonSize === 'small') return 'btn-sm'
```

Comment:
* [ ] FYI Consistency: For the `buttonColor` we store the bootstrap classes which saves us from helpers like this. We could do the same with `buttonSize`.